quickwit/quickwit-doc-mapper/src/routing_expression/mod.rs

```diff
@@ -26,7 +26,8 @@ use serde_json::Value as JsonValue;
 use siphasher::sip::SipHasher;
 
 pub trait RoutingExprContext {
-    fn hash_attribute<H: Hasher>(&self, attr_name: &str, hasher: &mut H);
+    fn hash_attribute<H: Hasher, F>(&self, attr_name: &str, hasher: &mut H, tranform: &F)
```

Comment:
```suggestion
    fn hash_attribute<H: Hasher, F>(&self, attr_name: &str, hasher: &mut H)
```

the transform stuff is not needed if you apply the modulo after the hash.